cpp/src/precompiled/arithmetic_ops.cc

```diff
@@ -155,12 +157,16 @@ NUMERIC_BOOL_DATE_FUNCTION(IS_NOT_DISTINCT_FROM)
 #define DIVIDE_NULL_INTERNAL(TYPE)                                                      \
   FORCE_INLINE                                                                          \
   TYPE divide_##TYPE##_##TYPE(TYPE in1, boolean is_valid1, TYPE in2, boolean is_valid2, \
-                              bool *out_valid) {                                        \
+                              int64 execution_context, bool* out_valid) {               \
     *out_valid = false;                                                                 \
     if (!is_valid1 || !is_valid2) {                                                     \
       return 0;                                                                         \
     }                                                                                   \
     if (in2 == 0) {                                                                     \
+      gandiva::helpers::ExecutionContext* execution_context_ptr =                       \
```

Comment:
this will become too verbose if we have to add this for every error fn. can we handle the cast and all that in the context fn ?

context_set_error(uint64_t context, const char *error) {}

Reply:
fixed.